docs/modules/ROOT/partials/nav.adoc

```diff
@@ -1,21 +1,102 @@
 * xref:index.adoc[Home]
+** xref:app-catalog:ROOT:explanations/what_is.adoc[What is it?]
+** xref:app-catalog:ROOT:explanations/why_exists.adoc[Why does it exist?]
+** xref:app-catalog:ROOT:explanations/app_catalog.adoc[Concept]
+** xref:app-catalog:ROOT:reference/glossary.adoc[Glossary]
+** xref:app-catalog:ROOT:reference/billing-naming.adoc[Naming Scheme for Billing]
+** xref:app-catalog:ROOT:reference/billing-appuio.adoc[Billing on APPUiO Cloud]
+** xref:app-catalog:ROOT:reference/service-maturity.adoc[Service Maturity]
+** xref:app-catalog:ROOT:reference/framework-requirements.adoc[Framework Requirements]
+** xref:app-catalog:ROOT:reference/quality-requirements.adoc[Quality Requirements]
+*** xref:app-catalog:ROOT:reference/quality-requirements/maintainability/readiness-standards.adoc[Maintainability]
+*** xref:app-catalog:ROOT:reference/quality-requirements/portability/backup-exports.adoc[Portability]
+*** Reliability
+**** xref:app-catalog:ROOT:reference/quality-requirements/reliability/automatic-updates.adoc[Automatic Updates]
+**** xref:app-catalog:ROOT:reference/quality-requirements/reliability/mandatory-updates.adoc[Mandatory Updates]
+**** xref:app-catalog:ROOT:reference/quality-requirements/reliability/backup-interval.adoc[Minimum Backup Interval]
+
+*** Usability
+**** xref:app-catalog:ROOT:reference/quality-requirements/usability/actionable-alerts.adoc[Actionable Alerts]
+**** xref:app-catalog:ROOT:reference/quality-requirements/usability/api-declarative.adoc[Declarative Configuration of Instances]
+**** xref:app-catalog:ROOT:reference/quality-requirements/usability/api-validation.adoc[Service Instance API Validation]
+**** xref:app-catalog:ROOT:reference/quality-requirements/usability/provisioning-time.adoc[Automated Provisioning of a Service]
+**** xref:app-catalog:ROOT:reference/quality-requirements/usability/logs.adoc[Service Instance Logs]
+
+* PostgreSQL
+** xref:app-catalog:ROOT:explanations/decisions/postgresql.adoc[]
+** xref:app-catalog:ROOT:explanations/decisions/postgres-monitoring.adoc[]
+** xref:app-catalog:ROOT:explanations/decisions/capacity-alerting.adoc[PostgreSQL by VSHN Capacity Alerts]
 
-.Applications
 * xref:redis.adoc[Redis]
+** xref:app-catalog:ROOT:explanations/decisions/redis.adoc[]
+** xref:app-catalog:ROOT:explanations/redis.adoc[Redis Service Details]
```

Comment:
That page contains a link labeled `Explanation → Redis`. This does no longer apply. Please update.

Reply:
Cannot reproduce. All resolved.